app/models/mixins/live_metrics_mixin.rb

```diff
@@ -1,8 +1,13 @@
 module LiveMetricsMixin
   extend ActiveSupport::Concern
 
+  LIVE_METRICS_DIR = Rails.root.join("product/live_metrics")
+
   class MetricValidationError < RuntimeError; end
 
+  delegate :metrics_available, :to => :metrics_capture
```

Comment:
delegate is needed as model classes that include live_metrics_mixin should provide a metrics_capture object with the specific implementation of metrics_available and collect_live_metric methods. These delegate were placed at model class but it is something it can go into the mixin for future re-use.
I'm not using alias_method as these methods are not at object level.